litgpt/model.py

```diff
@@ -244,7 +244,7 @@ def __init__(self, config: Config, block_idx: int) -> None:
         super().__init__()
         shape = (config.n_head + 2 * config.n_query_groups) * config.head_size
         # key, query, value projections for all heads, but in a batch
-        self.attn = nn.Linear(config.n_embd, shape, bias=config.bias)
+        self.attn = nn.Linear(config.n_embd, shape, bias=config.bias or config.attn_bias)
```

Comment:
This logic needs to be implemented in `__post_init__` of the Config class.

Here `bias` value should be controlled only by `config.attn_bias`.

Reply:
I think it's fine for now.
If we need to control bias in more places, I guess we can revive #878